app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -75,7 +76,33 @@ class HomeScreenRobot {
 
     fun verifyExistingTabList() = assertExistingTabList()
 
-    private fun scrollToElementByText(text: String): UiScrollable {
+    // Collections element
+    fun clickCollectionThreeDotBtn() {
```

Comment:
For consistency, we should probably used "Button" or "Btn" in all our tests, but not both.  I actually think this shorter version is still intuitive and shorter, but then we should prob also change it in our other tests as well (at some point)